src/command/cmd_funcs.c

```diff
@@ -9397,14 +9397,23 @@ cmd_slashguard(ProfWin* window, const char* const command, gchar** args)
     return TRUE;
 }
 
+gchar*
+_url_make_id()
+{
+    gchar* id = malloc(6 * sizeof(char));
```

Comment:
If you need to allocate memory and call sprintf you should use g_strdup_printf() instead :)